### ISSUE TITLE
Wine tray icons don't seem to work at all past displaying an icon.

### ISSUE THREAD

**Issue by queenbiscuit311** [ubuntu/gnome-shell-extension-appindicator#481]
This extension is great, but it just seems to be completely non-functional for wine apps. Their icons show up (although they have black where there should be transparency), but right clicking does nothing. This is the case for all wine apps. I know this is an issue with this extension and not wine, because every other DE I've used with appindicator support works with them as expected. In addition, some wine apps with multiple tray icons will only show one. 

DE is GNOME 45 on Wayland.

**Comment by 238Ver** [comment]
Yes, the same issue. The wine tray in the panel is unclickable. So if I close the wine APP window, I have to restart the APP to show the interface.

**Comment by queenbiscuit311** [comment]
> Yes, the same issue. The wine tray in the panel is unclickable. So if I close the wine APP window, I have to restart the APP to show the interface.

you can at least work around this by disabling legacy icon support in the extension settings. it sucks because then it just uses wines tray icon window but at least you can actually use the apps unlike before

**Comment by 238Ver** [comment]
Thx, it is work for me. And looking forward to a better experience.
